### PR TITLE
Fixes monitors, adds radials to them

### PR DESCRIPTION
**Pull request by quardbreak**
## Changelog
:cl:
bugfix: Fixes monitor mask initial world icon.
rscadd: Adds radial icon choice for monitors.
/:cl:

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
This is going to conflict with the monitor conversion in #1112.

**Comment by quardbreak** [comment]
This breaks on mob icon. Need to fix this.

**Comment by quardbreak** [comment]
Fixed

File: code/modules/clothing/masks/monitor.dm [hunk 1]
```diff
@@ -69,28 +75,38 @@
...
-	var/choice = input("Select a screen icon.") as null|anything in monitor_states
+
+	var/list/options = list()
+	for(var/i in monitor_states)
+		var/image/radial_button = image(icon, icon_state = "[initial(icon_state)]-[monitor_states[i]]")
+		radial_button.name = i
+		options[i] = radial_button
+
+	var/choice = show_radial_menu(usr, usr, options, radius = 42, require_near = TRUE, tooltips = TRUE)
 	if(choice)
```
Comment (MistakeNot4892) [inline code/modules/clothing/masks/monitor.dm]: Need to check if the monitor has been qdeleted, if it's still in the user's inv, if the user is incapacitated, etc.

File: code/modules/clothing/masks/monitor.dm [hunk 2]
```diff
@@ -69,28 +75,38 @@
...
 		var/obj/item/organ/external/E = user.organs_by_name[BP_HEAD]
 		if(istype(E) && BP_IS_PROSTHETIC(E))
-			return 1
+			return TRUE
 		to_chat(user, SPAN_WARNING("You must have a synthetic head to install this upgrade."))
-	return 0
+	return FALSE
+
+/obj/item/clothing/mask/monitor/attack_self(mob/user)
+	set_monitor_state()
```
Comment (MistakeNot4892) [inline code/modules/clothing/masks/monitor.dm:87]: I believe attack_self needs to return TRUE.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**